Extension/package.json

```diff
@@ -1467,6 +1412,9 @@
       },
       "[c]": {
         "editor.wordBasedSuggestions": false
+      },
+      "[Log]": {
+        "editor.wordWrap": "off"
```

Comment:
Does this turn it off for all logs?

Reply:
Yeah, I filed a bug on VS Code to improve the scenario. Users who really don't like that can overwrite the default. From the all the logs I've seen, wordwrapping is not important.

Reply:
See https://github.com/microsoft/vscode/issues/78245